lib/suse_container_urls.pm

```diff
@@ -0,0 +1,76 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Database for URLs of docker images to be tested
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+package suse_container_urls;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap);
+
+our @EXPORT = qw(
+  get_suse_container_urls
+);
+
+# Returns a tuple of image urls and their matching released "stable" counterpart.
+# If empty, no images available.
+sub get_suse_container_urls {
+    my $version    = get_required_var('VERSION');
+    my $dotversion = $version =~ s/-SP/./r;         # 15 -> 15, 15-SP1 -> 15.1
+    $dotversion = "${dotversion}.0" if $dotversion !~ /\./;    # 15 -> 15.0
+
+    my @image_names  = ();
+    my @stable_names = ();
+    if (is_sle(">=12-sp3")) {
+        my $lowerversion  = lc $version;
+        my $nodashversion = $version =~ s/-//r;
```

Comment:
This is still failing for me, see my [test run](http://pdostal-server.suse.cz/tests/1072#step/docker_image/31)
```
# Test died: command 'docker pull registry.suse.de/suse/sle-12-sp3/docker/update/cr/images/suse/sle-12SP3' failed at /var/lib/openqa/share/tests/sle/tests/console/docker_image.pm line 55.
invalid reference format: repository must be lowercase
```
```suggestion
        my $nodashversion = $version =~ s/-SP/sp/r;
```